Client/Frontend/Fakespot/FakespotViewModel.swift

```diff
@@ -0,0 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class FakespotViewModel {
+    // Define your properties and methods for the ViewModel here
```

Comment:
Just curious what we plan to put here? I guess it's in the coming tasks?

Reply:
Correct, view model logic will be handled by [FXIOS-6976](https://mozilla-hub.atlassian.net/browse/FXIOS-6976)